netfields/protocols.py

```diff
@@ -0,0 +1,46 @@
+from ipaddress import ip_interface, IPv4Interface, IPv6Interface, ip_network, IPv4Network, IPv6Network, ip_address, \
+    IPv4Address, IPv6Address
+
+BOTH_PROTOCOLS = 'both'
+IPV4_PROTOCOL = 'IPv4'
+IPV6_PROTOCOL = 'IPv6'
+PROTOCOLS = [BOTH_PROTOCOLS, IPV4_PROTOCOL, IPV6_PROTOCOL]
```

Comment:
Technically, since we are dealing with addresses and not packets, this should be "address family" not "protocol". Also, why not use the standard definitions that already exist and are already used in functions involving addresses: AF_INET, AF_INET6, and AF_UNSPEC?

Reply:
I was thinking about functions in the standard library, which tend to use the AF_* constants, but if I look at the ipaddress module itself, which is what this module is intended to use, the objects have a property "version" which takes an integer. Perhaps that would be better for consistency here. Then perhaps None for unspec.

Reply:
@jimfunk Sorry for the late response, I was looking at it now and it seems like the just use the number and this is prone to problems in case of a change. The options I see is to use the `_version` field even though its protected or create an instance and use its `version` property. What do you think?

Reply:
Simply changed it to numbers and None as in ipaddress since the package itself does not export version types.